lib/utils.js

```diff
@@ -25,8 +25,13 @@ var nodes = require('./nodes')
  */
 
 exports.absolute = function(path){
-  // On Windows the path could start with a drive letter, i.e. a:\\ or two leading backslashes
-  return path.substr(0, 2) == '\\\\' || '/' === path.charAt(0) || /^[a-z]:\\/i.test(path);
+  // On Windows the path could start with a drive letter, i.e. a:\\ or two leading backslashes.
+  // Also on Windows, the drive letter can be capital, and the path may have been normalized to 
+  // forward slashes, so check for this too.
+  return path.substr(0, 2) == '\\\\'
+      || '/' === path.charAt(0)
+      || /^[a-zA-Z]:\\/i.test(path)
```

Comment:
Last two checks could be simplified to `/^[a-z]:[\\\/]/i` (p.s. the `i` flag already means that this check is case-insensitive).